service/src/main/kotlin/xyz/lebkuchenfm/external/storage/dropbox/models/DropboxFileUpload.kt

```diff
@@ -9,6 +9,7 @@ data class DropboxFileUploadArgs(
     val mode: String,
     val autorename: Boolean,
     val mute: Boolean,
+    @SerialName("strict_conflict") val strictConflict: Boolean,
```

Comment:
Wait, why `@SerialName` here? What does it do? Rename from snake_case to camelCase?

Reply:
Yup. Formatter (inteliJ?) complains at snake case but thats how it is in api.

Reply:
Funny that it's the same annotation for field rename and type discrimination.